cylc/flow/etc/examples/datetime-cycling/flow.cylc

```diff
@@ -0,0 +1,41 @@
+[meta]
+    title = Integer Cycling
+    description = """
+        A basic cycling workflow which runs the same set of tasks over
+        and over. Each cycle will be given a datetime identifier.
+
+        The task "a" will wait until the real-world (or wallclock) time passes
+        the cycle time.
+
+        Try changing the "initial cycle point" to "previous(00T00) - P1D" to
+        see how this works.
+    """
+
+[scheduling]
+    # start this workflow today at midnight
+    initial cycle point = previous(T00)
+
+    [[graph]]
+        # run this with a "P"eriod of "1" "D"ay -> P1D
+        P1D = """
+            # this is the workflow we want to repeat:
+            a => b => c & d
+
+            # this is an "inter-cycle dependency", it makes the task "b"
+            # wait until its previous instance has completed:
+            b[-P1D] => b
+
+            # this makes the task "a" wait until the cycle point matches
+            # the real world time - i.e. this makes the workflow run in
+            # "real time", to run in "simulated time" remove this line:
+            @clock => a
```

Comment:
> " this makes the workflow run in "real time"

Unless it is running behind. This might be too verbose:

```suggestion
            # the real world time - i.e. it prevents the workflow from getting
            # ahead of the clock. If the workflow is running behind (e.g. after
            # a delay, or from an earlier initial cycle point) it will catch
            # until the clock-trigger constrains it again. To run entirely in
            # "simulated time" remove this line:
            @clock => a
```